content/en/observability_pipelines/sources/azure_event_hubs.md

```diff
@@ -0,0 +1,176 @@
+    1.  For **Kafka SASL Password**, enter the full connection string (for example, `Endpoint=sb://<NAMESPACE>.servicebus.windows.net/;SharedAccessKeyName=<PolicyName>;SharedAccessKey=<Key>`).
+        - This is the **Primary Connection String** in your Event Hub instance [shared access policies](#configure-shared-access-policy).
+    1. Enter your Kafka TLS passphrase.
+        - This is the **Primary Key** in your Event Hub instance's [shared access policies](#configure-shared-access-policy).
+    {{< img src="observability_pipelines/sources/kafka_env_vars.png" alt="The install page with example values for the kafka environment variables" style="width:100%;" >}}
+1. Enter the environment variables for your destinations, if applicable.
+1. Follow the rest of the instructions on the page to install the Worker based on your platform.
+{{% /tab %}}
+{{% tab "Virtual machine (VM)" %}}
+
+1. Navigate to [Observability Pipelines](https://app.datadoghq.com/observability-pipelines).
+1. Select the Kafka source.
+    1.  In the **Group ID** field, specify or create a unique consumer group (for example, `datadog-consumer-group`).
+    1.  Enter `datadog-topic` in the **Topics** field.
+    1.  Toggle the switch to enable SASL authentication.
+    1.  In the **Mechanism** dropdown menu, select **PLAIN**.
+    1.  Enable TLS. For the certificate, copy the certificate from its original location to the default Observability Pipelines data configuration directory:
+        1. Since the Observability Pipelines Worker hasn't been installed yet, run this command to create the directory for the certificate:
+            ```
+            sudo mkdir -p /var/lib/observability-pipelines-worker/config
+            ```
+        1. Run this command to copy the certificate to the directory you created:
+            ```
+            sudo cp /etc/ssl/certs/ca-certificates.crt /var/lib/observability-pipelines-worker/config/
+            ```
+        1. In the **Certificate path** field, enter `/cert.pem`.
+    {{< img src="observability_pipelines/sources/kafka_settings.png" alt="The Kafka source settings with example values" style="width:45%;" >}}
+1. Click **Next: Select Destination**.
+1. After you set up your destinations and processors, click **Next: Install**.
+1. Select your platform in the **Choose your installation platform** dropdown menu.
+1. Enter the environment variables for your Kafka source:
+    1.  For **Kafka Bootstrap Servers**, enter `<NAMESPACE>.servicebus.windows.net:9093` (for example, `myeventhubns.servicebus.windows.net:9093`).
+    1.  For **Kafka SASL Username**, enter `\$\$ConnectionString`. **Note**: You must escape the `$` in front of `ConnectionString`, otherwise the environment variable won't be loaded.
+    1.  For **Kafka SASL Password**, enter the full connection string (for example, `Endpoint=sb://<NAMESPACE>.servicebus.windows.net/;SharedAccessKeyName=<PolicyName>;SharedAccessKey=<Key>`).
```

Comment:
Do we want to note this needs to be wrapped in quotes? https://github.com/DataDog/logs-psa-private/tree/main/POCs-Opps-hacks/k8s-op-certificates#virtual-machine-setup:~:text=DD_OP_SOURCE_KAFKA_SASL_PASSWORD%20must%20be%20wrapped%20in%20quotes